pkg/podlister/resolve_container.go

```diff
@@ -284,3 +283,14 @@ func extractPodContainerIDfromPath(path string) (string, error) {
 	}
 	return systemProcessName, fmt.Errorf("failed to find pod's container id")
 }
+
+// GetAlivePods returns pod list
+func GetAlivePods() map[string]bool {
+	pods, _ := podLister.ListPods()
```

Comment:
a bit paranoid, if the err actually happens, the `pods` is emtyp, and `alivePods` is empty. Then the caller may get the wrong impression and think all Pods are inactive.

Reply:
agree. just add error handling change.